src/sinks/datadog/logs/service.rs

```diff
@@ -95,20 +95,33 @@ pub struct LogApiService {
     client: HttpClient,
     uri: Uri,
     user_provided_headers: IndexMap<HeaderName, HeaderValue>,
+    default_headers: IndexMap<HeaderName, HeaderValue>,
 }
 
 impl LogApiService {
     pub fn new(
         client: HttpClient,
         uri: Uri,
         headers: IndexMap<String, String>,
+        dd_evp_origin: String,
     ) -> crate::Result<Self> {
-        let headers = validate_headers(&headers)?;
+        let user_provided_headers = validate_headers(&headers)?;
+
+        // Note that these headers cannot be overriden by the user.
```

Comment:
Is this documented?

Reply:
No. I'm not sure if we'd want to document this. The user should never want to modify these values unless for some nefarious reasons.